tasks/installation.yml

```diff
@@ -211,6 +211,11 @@
   become_user: "{{ magento_user }}"
   command: "/usr/bin/php {{ magento_app_root }}/bin/magento cache:flush"
 
+- name: "Let Magento indexer run"
```

Comment:
Isn't this only needed on fresh instances? Maybe it should be somehow conditional after all on huge databases and multishop systems this can take a while to complete

Reply:
valid concern, I was thinking the same.

Will think about some smart condition here.

Reply:
adjusted